caas/kubernetes/provider/bootstrap.go

```diff
@@ -704,7 +707,7 @@ func (c *controllerStack) ensureControllerServiceAccount() error {
 		return errors.Trace(err)
 	}
 
-	crb := &rbacv1.ClusterRoleBinding{
+	crb := resources.NewClusterRoleBinding(c.broker.GetCurrentNamespace(), &rbacv1.ClusterRoleBinding{
```

Comment:
we have
`c.broker.GetCurrentNamespace(), // name cluster role binding after the controller namespace.`
below.
Maybe assign to a var with the comment and use the var